docs/modules/ROOT/pages/java.adoc

```diff
@@ -857,13 +830,16 @@ explicitly enable them in the load balancer's configuration (see
 https://developer.couchbase.com/documentation/mobile/1.5/guides/sync-gateway/nginx/index.html[Load
```

Comment:
Current documentation link for the Load Balancer should be as below.
https://developer.couchbase.com/documentation/mobile/current/guides/sync-gateway/nginx/index.html